packages/react/spec/imports/named-esm-import.mjs

```
@@ -0,0 +1 @@
+import { Provider } from "../../dist/esm/index.js";
```

Should we also do a `throw` if `Provider` is undefined, or does ESM give us some guarantees around that?

---

It will do it for us, I experienced that when putting this together

---

Just to double check: do you think there's a scenario where we accidentally have `export const Provider = ...something evaluating to undefined...`? If not, 🚀